sim/warrior/deep_wounds.go

```diff
@@ -23,7 +23,7 @@ func (warrior *Warrior) applyDeepWounds() {
 		ActionID:       core.ActionID{SpellID: spellID},
 		SpellSchool:    core.SpellSchoolPhysical,
 		ProcMask:       core.ProcMaskEmpty,
-		Flags:          core.SpellFlagNoOnCastComplete | core.SpellFlagPassiveSpell,
+		Flags:          core.SpellFlagIgnoreAttackerModifiers | core.SpellFlagNoOnCastComplete | core.SpellFlagPassiveSpell,
```

Comment:
Deep wounds did big dum dum damage without this flag because we have to use the attacker multiplier function now and it doesn't benefit from unit multipliers